cmd/influx/cli/cli.go

```diff
@@ -729,9 +730,31 @@ func (c *CommandLine) ExecuteQuery(query string) error {
 		}
 		query = pq.String()
 	}
-	response, err := c.Client.Query(c.query(query))
+
+	ctx := context.Background()
+	if !c.IgnoreSignals {
+		done := make(chan struct{})
+		defer close(done)
+
+		var cancel func()
+		ctx, cancel = context.WithCancel(ctx)
+		go func() {
+			select {
+			case <-done:
+			case <-c.osSignals:
+				cancel()
+			}
+		}()
+	}
+
+	response, err := c.Client.QueryContext(ctx, c.query(query))
 	if err != nil {
-		fmt.Printf("ERR: %s\n", err)
+		if err.Error() == "" {
+			fmt.Println("Aborting Query..")
```

Comment:
This shouldn't print aborting query. You should do:

```
if err := ctx.Err(); err == context.Canceled {
    return errors.New("aborted by user")
} else {
    return err
}
```

Then the normal error printing loop will print the correct message.